jo.c

```diff
@@ -47,11 +47,35 @@
 static JsonNode *pile;		/* pile of nested objects/arrays */
 
 #if defined(_WIN32) || defined(_AIX)
-# define err(n, s)	{ fprintf(stderr, s); exit(n); }
-# define errx(n, f, a)	{ fprintf(stderr, f, a); exit(n); }
+#include <errno.h>
+#include <stdarg.h>
```

Comment:
If wanted, I can move these includes up to an else branch for the `#if !defined(WIN32) && !defined(_AIX)` condition. I've only included them here for no particular reaso